package.json

```diff
@@ -97,48 +98,47 @@
   "scripts": {
     "_eslint": "eslint --config config/eslint.js --report-unused-disable-directives",
     "_prettier": "prettier . --ignore-path .gitignore",
-    "audit": "npm run audit:deprecations && npm run audit:vulnerabilities",
+    "audit": "npm-run-all audit:*",
     "audit:deprecations": "depreman --errors-only",
     "audit:vulnerabilities": "better-npm-audit audit",
     "audit:vulnerabilities:runtime": "better-npm-audit audit --production",
+    "check": "npm-run-all check:*",
+    "check:ci": "node script/maybe-run.js actionlint",
+    "check:dependencies": "knip --config config/knip.jsonc",
+    "check:formatting": "npm run _prettier -- --check",
+    "check:js": "npm run _eslint -- '**/*.{cjs,js}'",
+    "check:json": "npm run _eslint -- '**/*.{json,jsonc}'",
+    "check:licenses": "licensee --errors-only",
+    "check:lockfile": "lockfile-lint",
+    "check:manifest": "npm run transpile && publint --strict && attw --pack . && ls-engines",
+    "check:md": "markdownlint --config config/markdownlint.yml --dot --ignore-path .gitignore .",
+    "check:sh": "node script/maybe-run.js shellcheck script/hooks/*.sh script/hooks/pre-*",
+    "check:yml": "npm run _eslint -- '**/*.yml'",
     "clean": "node script/clean.js",
-    "coverage": "npm run coverage:unit && npm run coverage:integration && npm run coverage:e2e && npm run coverage:compat && npm run coverage:breakage",
-    "coverage:breakage": "c8 --config config/c8/breakage.json npm run test:breakage",
-    "coverage:compat": "c8 --config config/c8/compat.json npm run test:compat",
-    "coverage:e2e": "node script/run-platform-coverage.js e2e",
-    "coverage:e2e:unix": "c8 --config config/c8/e2e-unix.json npm run test:e2e",
-    "coverage:e2e:win": "c8 --config config/c8/e2e-win.json npm run test:e2e",
+    "coverage": "npm-run-all coverage:*",
+    "coverage:unit": "c8 --config config/c8/unit.json npm run test:unit",
     "coverage:integration": "node script/run-platform-coverage.js integration",
     "coverage:integration:unix": "c8 --config config/c8/integration-unix.json npm run test:integration",
     "coverage:integration:win": "c8 --config config/c8/integration-win.json npm run test:integration",
-    "coverage:unit": "c8 --config config/c8/unit.json npm run test:unit",
+    "coverage:e2e": "node script/run-platform-coverage.js e2e",
+    "coverage:e2e:unix": "c8 --config config/c8/e2e-unix.json npm run test:e2e",
+    "coverage:e2e:win": "c8 --config config/c8/e2e-win.json npm run test:e2e",
+    "coverage:compat": "c8 --config config/c8/compat.json npm run test:compat",
+    "coverage:breakage": "c8 --config config/c8/breakage.json npm run test:breakage",
```

Comment:
For the record: these (as well as `mutation:*` and `test:*`) were re-ordered because `npm-run-all coverage:*` runs these commands in the order in which they're declared, and we want to run unit tests, then integration tests, then end-to-end tests, etc.